### ISSUE TITLE
extra760 results in success despite errors

### ISSUE THREAD

**Issue by dlorch** [toniblyx/prowler#940]
`lambda:GetFunction` returns a permission denied error, but the function returns a `PASS`. See logs below (Account ID masked, but otherwise log output captured as-is).

```
7.60 [extra760] Find secrets in Lambda functions code - lambda [Critical] 
An error occurred (AccessDeniedException) when calling the GetFunction operation: User: arn:aws:sts::1234567890:assumed-role/ProwlerXA-CBRole/ProwlerAssessmentSession is not authorized to perform: lambda:GetFunction on resource: arn:aws:lambda:eu-central-1:1234567890:function:CustomControlTowerDeploymentLambda because no identity-based policy allows the lambda:GetFunction action 
curl: no URL specified! 
curl: try 'curl --help' or 'curl --manual' for more information 
unzip: cannot find or open ./prowler/secrets-231541534269/extra760-CustomControlTowerDeploymentLambda-eu-central-1/CustomControlTowerDeploymentLambda-code.zip, ./prowler/secrets-231541534269/extra760-CustomControlTowerDeploymentLambda-eu-central-1/CustomControlTowerDeploymentLambda-code.zip.zip or ./prowler/secrets-231541534269/extra760-CustomControlTowerDeploymentLambda-eu-central-1/CustomControlTowerDeploymentLambda-code.zip.ZIP. 
 PASS! eu-central-1: No secrets found in Lambda function CustomControlTowerDeploymentLambda code 
```

**Comment by toniblyx** [comment]
Thanks @dlorch for the catch. Yes, that error must be handled better for sure.

**Comment by toniblyx** [comment]
@dlorch this is now fixed in devel branch 2.7 if you want to test it.

**Comment by dlorch** [comment]
Awesome, thanks!
